build/RunTestsOnHelix.sh

```diff
@@ -18,5 +18,12 @@ cp -a $HELIX_CORRELATION_PAYLOAD/t/TestExecutionDirectoryFiles/. $TestExecutionD
 dotnet new --debug:ephemeral-hive
 # We downloaded a special zip of files to the .nuget folder so add that as a source
 dotnet new nugetconfig
-dotnet nuget add source $DOTNET_ROOT/.nuget --configfile nuget.config
-dotnet nuget list source
+dotnet nuget add source $DOTNET_ROOT/.nuget --configfile $TestExecutionDirectory/nuget.config
```

Comment:
Is this correct? Is this the unpacked, prepolated feed or the nuget cache?

Reply:
If it's getting added as a source it should be the flat folder of nupkgs.

Reply:
You are correct that the current zips are a cache rather than a feed and this change won't help. I can wait to make this change until I have fixed the zip to be the flat list. I made it now as I was already removing feeds from the file so wanted it to be consistent but you're correct that it's not relevant yet.